pkg/api/util/clusteragent.go

```diff
@@ -0,0 +1,184 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package util
```

Comment:
Could we rename this to clusteragent ? To be consistent with the other utils in the pkg ?

Reply:
I agree with that change, @DataDog/kenafeh any thoughts ?

Reply:
indeed. ideally, it should have it's build tags as it's not needed for the puppy agent (for the moment?)